src/ir_Daikin.cpp

```diff
@@ -1287,7 +1287,7 @@ std::string IRDaikin2::toString() {
     result += IRDaikinESP::renderTime(getSleepTime());
   else
     result += F("Off");
-	result += F(", Beep: ");
+    result += F(", Beep: ");
```

Comment:
Need to reduce the indent by one level for these two statements (i.e. remove two spaces the _beep_ lines)

Reply:
Fixed.